simplq/src/components/pages/Join/Form.jsx

```diff
@@ -68,9 +68,16 @@ export function JoinQueueForm(props) {
     });
   };
 
-  const JoinButton = () => (
-    <div>{addingInProgress ? <LoadingIndicator /> : <JoinQButton onClick={handleClick} />}</div>
-  );
+  const ActionButton = () => {
```

Comment:
we have to pass props here.

Reply:
It's using the JoinQueueForm's props here, so no need.